### ISSUE TITLE
ProtocolLib WrappedServerPing Error

### ISSUE THREAD

**Issue by Acult** [aadnk/ProtocolLib#164]
_Follow this template except for feature requests. Use pastebin when providing /protocol dump and any relevant errors._

Make sure you've done the following:
- [YES] You're using the latest build for your server version
- [YES] This isn't an issue caused by another plugin
- [YES] You've checked for duplicate issues
- [YES] You didn't use `/reload`

Debug paste link:
https://pastebin.com/EjeTwCJY

Description and relevant errors:
Basically, this errors happens to each plugiin which tries to modify something with WrappedServerPing. The errors is spammed and causes big lag spikes.
Server is 1.7.10 Thermos, and i'm using latest build of Protocol Lib for that version, i hope you will fix this since it's a very disturbing bug..
